test/federated/test_plan.py

```diff
@@ -255,7 +297,50 @@ def my_plan(data):
     x_ptr = x.send(socket_pipe)
     plan_res = plan_ptr(x_ptr).get()
 
-    assert (plan_res == th.tensor([-42, 24, 46])).all()
+    assert (plan_res == local_res).all()
+
+    # delete remote object before websocket connection termination
+    del x_ptr
+
+    server.terminate()
+    hook.local_worker.is_client_worker = True
+
+
+def test_execute_plan_module_remotely(hook, start_proc):
+    """Test plan execution remotely."""
+    hook.local_worker.is_client_worker = False
+
+    class Net(nn.Module):
+        def __init__(self):
+            super(Net, self).__init__()
+            self.fc1 = nn.Linear(2, 3)
+            self.fc2 = nn.Linear(3, 2)
+
+        @sy.method2plan
+        def forward(self, x):
+            x = F.relu(self.fc1(x))
+            x = self.fc2(x)
+            return F.log_softmax(x, dim=0)
+
+    net = Net()
+    # net.send(hook.local_worker)
```

Comment:
why are these commented out

Reply:
nice catch, these should actually be removed since they aren't needed anymore. Before these fixes this line was needed. thanks!